src/parser.rs

```diff
@@ -13,6 +13,8 @@ use library::*;
 type Reader = EventReader<BufReader<File>>;
 type Attributes = Vec<OwnedAttribute>;
 
+const EMPTY_CTYPE: &'static str = "/*EMPTY*/";
```

Comment:
This is obviously suboptimal but I'm not sure if it's better to have two `Parameter` types or put up with optional `c_type`. Perhaps the composition&`Deref` trick could help here too.